recipe/meta.yaml

```diff
@@ -57,7 +57,6 @@ test:
     - ptdump -h
     - ptrepack -h
     - pttree -h
-    - pip check
```

Comment:
We apply a patch at build time to remove the blosc2 python package form the dependency list (in `pyproject.toml`), because the conda package directly uses the c-blosc2 library.
In my understanding pip check uses the original `pyproject.toml` and of course detects the missing blosc2 (python package).

Reply:
Can you add a comment there so we can track this? We are ok removing pip-check in special cases but we need to remember why we did it.

Reply:
Good idea.
Comment added.
Please feel free to merge if you are happy with it.